data/learningpaths.yaml

```diff
@@ -0,0 +1,87 @@
+- id: gwapi
+  title: Gateway API
+  description: Learn the Kubernetes Gateway API with kgateway
```

Comment:
Possibly punctuate all the descriptions, but I'm fine either way since it's consistent.

Reply:
I was thinking that addition of a conventional period at the end can live on the hugo template so it doesn't have to be added to each and every description in the yaml.  what do you think?

Reply:
sounds good, I'll add it to the learning-path.html shortcode